bpaingest/util.py

```diff
@@ -57,9 +57,9 @@ def _register(fn):
     return _register, registered
 
 
-def make_logger(name):
+def make_logger(name, level=logging.DEBUG):
     logger = logging.getLogger(name)
-    logger.setLevel(logging.DEBUG)
+    logger.setLevel(level)
     handler = logging.StreamHandler()
```

Comment:
what's the intention here? the setLevel stuff just sets the default level, and you only ever seem to passing in `logging.DEBUG`, so this change doesn't seem to do anything?

Reply:
So often here, once I had my head around  lower-level stuff with DEBUG, there's then too much noise for doing an overall run or just honing in on a specific part, so I'd actually set this default to INFO locally - I only switched back to DEBUG as I didn't want to mess with usual behaviour.

Reply:
I didn't always want DEBUG set for everything either, allowed me to be more granular with certain parts I wasn't following without a DEBUG

Reply:
Personally I prefer leaving stuff in default of INFO, but yeah just kept it to DEBUG as this is what it started with.

Reply:
well, I think it'd be better then to add a CLI argument to set the log level globally rather than having to change the code to change the log level?